app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -36,11 +36,19 @@ div.panel.panel-default
         tr
           th = @submission.class.human_attribute_name(:grader)
           td
-            - @submission.graders.uniq.each do |grader|
-              = display_course_user(grader)
+            - if @submission.publisher
+              = display_course_user(@submission.publisher)
+            - else
+              - @submission.graders.uniq.each do |grader|
+                = display_course_user(grader)
         tr
           th = @submission.class.human_attribute_name(:graded_at)
-          td = format_datetime(@submission.graded_at) if @submission.graded_at
+          td
+            - if @submission.published_at
+              = format_datetime(@submission.published_at)
+            - else
```

Comment:
I think same here. For auto graded assessments, publisher will be set to the system.